docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2021-04-30
+ - Alert_on_Unexpected_Content_Types.js > Added Content-Type text/yml to the list of expected types.
```

Comment:
Just noticed it should be `yaml` (missing the a).

Reply:
Fixed